src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/Dashboard_TripSearchScreen.tsx

```diff
@@ -211,222 +210,225 @@ export const Dashboard_TripSearchScreen: React.FC<RootProps> = ({
 
   return (
     <View style={style.container}>
-      <FullScreenHeader
-        title={t(TripSearchTexts.header.title)}
-        rightButton={{type: 'chat'}}
-        leftButton={{
-          type: 'back',
-          onPress: () => {
-            if (callerRoute?.name) {
-              navigation.setParams({
-                callerRoute: undefined,
-              });
-
-              return navigation.navigate({
-                name: callerRoute?.name as any,
-                params: {},
-              });
-            }
+      <FullScreenView
+        headerProps={{
+          title: t(TripSearchTexts.header.title),
+          rightButton: {type: 'chat'},
+          leftButton: {
```

Comment:
This is more a design question, should we have the arrow icon here to stay consistent with other pages?

Reply:
I'll just keep it as is for now, as I understood that they only wanted the added parallax scroll and not any other header changes.